bridge/api/api.go

```diff
@@ -83,7 +110,14 @@ func (b *API) Send(msg config.Message) (string, error) {
 	if msg.Event == config.EventMsgDelete {
 		return "", nil
 	}
-	b.Messages.Enqueue(&msg)
+	b.Log.Debugf("enqueueing message from %s on ring buffer", msg.Username)
+	b.Messages.Enqueue(msg)
+
+	bytes, err := json.Marshal(msg)
```

Comment:
rename bytes to data so we don't have a collision with the bytes package

Reply:
done